CodeStarterCamp_Week1/main.swift

```diff
@@ -8,5 +8,20 @@
 
 import Foundation
 
-print("Hello, World!")
+func ice() {
+    for _ in 1...8{
+        print("***********")
+    }
+}
+
+func bar(){
```

Comment:
```suggestion
func bar() {
```
코드 컨벤션을 전체적으로 통일시켜주시면 좋을 것 같아요 😎